package.json

```diff
@@ -28,7 +28,30 @@
         "start": "npm-run-all build:sass --parallel watch:*",
         "build": "npm-run-all install:playground build:sass build:eleventy build:webpack images",
         "lint": "eslint --ext=.js,.jsx .",
-        "fix": "npm run lint -- --fix"
+        "fix": "npm run lint -- --fix",
+        "lint:scss": "stylelint '**/*.scss'",
+        "lint:scss:fix": "stylelint '**/*.scss' --fix",
```

Comment:
Please try to re use the lint command with additional arg like `lint`. ie: `npm run lint:scss -- --fix` so that next time if we change the directories with scss to lint we don't want to change the fix command too?

Reply:
done